src/main/java/uk/ac/cam/cl/dtg/segue/api/GroupsFacade.java

```diff
@@ -301,7 +301,7 @@ public Response getGroupsForGivenUser(@Context final HttpServletRequest request,
 
             RegisteredUserDTO userOfInterest = userManager.getUserDTOById(userId);
 
-            List<UserGroupDTO> groups = groupManager.getGroupsByOwner(userOfInterest);
+            List<UserGroupDTO> groups = groupManager.getAllGroupsOwnedAndManagedByUser(userOfInterest, false);
```

Comment:
The API doc said this got all groups owned or managed by the user, but it did not. We do not use this endpoint anywhere in the frontend at present, so it may be a candidate for removal.